cli/command/run/cmd.go

```diff
@@ -122,6 +124,7 @@ func Command(cli *command.SauceCtlCli) *cobra.Command {
 // Run runs the command
 func Run(cmd *cobra.Command, cli *command.SauceCtlCli, args []string) (int, error) {
 	println("Running version", version.Version)
+	checkForUpdates()
```

Comment:
Hm. I'm wondering if we should reduce the frequency of checks. Instead of every time, we could do 20-25% of the time?

Inside of `checkForUpdates()`:
```go
	if rand.Intn(100) > 25 {
		return
	} 
```

Reply:
Wouldn't it be weird in case you get the message, then on the next run you don't have the notification ?

I'm ok with checking only 1/4 of times, but we should consider saving the latest available version locally.

So, as soon as the message has been shown once, it will be always present.

Reply:
Well... I'm not strongly for or against it 😅 Can do the check every single time and see how well that performs over time!

Reply:
👍